### PR TITLE
Week 2

### PR DESCRIPTION
**Pull request by sarah-lenz**
With this pull request come changes to the UI of the app
- new features have been added
- the stimulation is now working
- the styling of the App was improved

I also started working on the final report (final_report.md), which can be found at the root of the directory.

### CODE REVIEW

**Review by Ilonka-v-Vliet — APPROVED** [review]
I can see you made a lot of progress, and based on the images used in final_report.md you got the grid working the way you wanted to as well! The documentation for the functions and for the app as a whole looks good. The explanation in the README.md for what each folder is used for is very helpful as well.

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
-tbd
\ No newline at end of file
+### Evolving grids
```
Comment (Bamske) [inline README.md:1]: Really clear readme file!!!

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,15 @@
+# for basic functionalities of app
```
Comment (Bamske) [inline requirements.txt:1]: Nice that you included notes explaining what each requirement is for!

**Review by Bamske — APPROVED** [review]
Hey Sarah, I see you made a lot of progress, and the GUI looks great! All functions have a discription with paramaters and you have a very informative readme file. Maybe for the functions you could include a usage example, but i'm not sure if that's really informative since they are not really accessed by the user. 
Great job!

**Pull request merged**